docs/kind.md

```diff
@@ -436,6 +438,36 @@ $ cd ../../contrib/
 $ KIND_IMAGE=example.com/kindest/node K8S_VERSION=v1.24.0 ./kind.sh
 ```
 
+### Using kind local registry to deploy non ovn-k containers
+
+If something extra need to be deployed and updated ovn-k kind can be started
+with 
+```
+./kind.sh --local-kind-registry
+```
+
+Then the images and manifests has to point to the prefix `localhost:5000` so 
+they can be push and deploy.
+
+
+### Loading ovn-kuberntes changes without restarting kind
+
+At some scenarios user may have other stuff apart from ovn-kubernetes at the
+kind cluster so restarting it involve redploying those.
+
+The kind.sh has the "--deploy" flag to refresh ovn-kubernets without restarting
+kind, so to reload the code just doing the following is enough:
+
+```bash
+# Default options will use kind mechanism to push images directly to the 
+# cluster
+./kind.sh --deploy
+
+# Using the kind local registry will use the connected local registry to
+# push images, making it easier to deploy non ovn-k artifacts.
```

Comment:
```
Using a local registry is an alternative to deploy ovn-kubernetes updates while also being useful to deploy other local images
```
I will admit though that I am missing the usefulness of hosting both ovn-k and other images in the local registry other than convenience. :thinking:

Reply:
The "kind load docker-image" is a kind ad-hoc solution that do not play well with projects that already use the typical "docker push", in fact kind team is planing on replacing it totally with local registry running at pod on the cluster [1]

[1] https://github.com/kubernetes-sigs/kind/issues/1213

Reply:
nit: could you replace `Using the kind local registry...` with my suggestion and remove `At some scenarios use...`?

Reply:
Done